doc/lua_api.md

```diff
@@ -654,9 +668,9 @@ on top of `cobble.png`.
 
 Parameters (all integers):
 
-* `<t>`: tile count (in each direction)
-* `<n>`: animation frame count
-* `<p>`: current animation frame (counting starts at 0)
+* `<t>`: tile count (in each direction) (integer [imagesize])
+* `<n>`: animation frame count (integer [imagesize])
+* `<p>`: current animation frame (integer [0, 2^31-1], counting starts at 0)
```

Comment:
should probably be imagesize as well, anything larger is always out of bounds

Reply:
Going by your logic it would actually be [0, 22999] instead, because counting starts at 0.